docs/build/identity.rs/1.0/docs/welcome.mdx

```diff
@@ -0,0 +1,92 @@
+
+There is a growth in applications that generate Digital Twins for physical devices or objects, such as the Asset Administration Shell (AAS) developed for our Industry Marketplace. Digital twins are online profiles representing a device or object. They provide a virtual state that mirrors reality by emulating the device or object’s physical state through data input sources like sensors. A digital twin is often used to monitor states and execute actions based on the information. Digital twins are only rarely shared outside the associated application and organization due to the complexities in sharing and matching profiles. However, empowered with a digital identity, digital twin sharing would become possible. Once data is verifiable and trusted, digital twins can form the basis for the digital representation of physical devices and objects. This allows other identities to interact with them automatically and provide services such as predictive maintenance.
+
+Security is a major barrier in advancing technologies that use IoT. Whether it is the smart devices in our own homes, or at a larger scale, the critical infrastructure of organizations and cities, security must be at the core. It is central to any globally-unifying identity solution. By integrating advanced research in cryptography and digital ledgers, and combining it with a scalable access and management system, security will become a core functionality of the systems we build. By using scalable device DIDs, integrating verification and reputation schemes, and allowing for transparent tamper-proof accountability, we begin to understand how we can future-proof the security of our systems, allowing us to start trusting the process, and not the patch.
+
+### One Framework. Any Identity
+
+The IOTA Identity framework serves as a ubiquitous layer of trust for the internet. Whether it's people, organizations, or things, the framework enables the creation of digital identities, fosters trust-building through verifiable credentials, and ensures seamless interaction among different entities.
+
+### Why IOTA?
+
+IOTA stands apart as a scalable, feeless Distributed Ledger Technology (DLT), suitable for a universal identity solution. Some features of IOTA include:
+
+1. **Permissionless & Decentralized**: IOTA ensures neutrality and resists censorship, as every participant is an integral part of the consensus.
+2. **Public**: IOTA's transparent nature ensures every transaction on the network is observable.
+3. **Feeless**: Both data and value transactions on IOTA's network are free of cost.
+4. **Open Source**: IOTA's codebase is publicly accessible and open to contributions.
+5. **Ease of use**: IOTA Identity is user-friendly and secure, thanks to its integration with the [Stronghold](/stronghold.rs/welcome) project.
```

Comment:
Maybe we can swap out some of these points for points more specific to Identity. The main reason you'd want to use IOTA for DID is because it acts as a global and trustless layer for persistence of and access to DID data at no cost for token holders.

Reply:
The talking points we came up with recently are:
Cost effectiveness: Usually minting decentralized identities costs fees. IOTA Identity has redeemable and predictable deposits, but no fees.
High availability: Identities are available on all nodes of the network, at all times -  for holders, issuers and verifiers. 
Security: Write-access to identities is secured through multi-level control structures with key rotation capabilities, allowing for backup access and recoverability.
Integrity: Updates go through the same mechanisms that secures the IOTA network, guaranteeing consistent state and history of all identities.